### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by danalvrz**
- Added media query for desktop screen size
- Implemented responsive web design principles

### CODE REVIEW

**Comment by danalvrz** [comment]
Global Webhint error continues

File: styles.css [hunk 1]
```diff
@@ -366,3 +423,458 @@ textarea::-webkit-input-placeholder {
...
+
+.project-list img {
+  transition: 0.7s;
+}
+
+.project-list .laptop:hover {
+  transform: scale(1.02);
+}
+
+@media (min-width: 768px) {
```
Comment (D4nielJ) [inline styles.css:435]: [OPTIONAL] I recommend you stick to the recommendation from Microverse on media queries, so definitively consider doing this before merging.

> Media Queries for the Desktop breakpoint in the CSS file. (the value for `min-width` property should be in range of `992px` - `1100px`)

File: index.html [hunk 2]
```diff
@@ -22,206 +22,216 @@
...
-          <input type="email" id="email" name="email" placeholder="Email" required>
-        </li>
-        <li>
-          <textarea placeholder="Enter your message here..." maxlength="500" required></textarea>
-        </li>
-        <li>
-          <button type="submit">Start collaboration</button>
-        </li>
-      </ul>
+        <ul>
```
Comment (D4nielJ) [inline index.html:199]: [OPTIONAL] I think you missed the default styles of your lists in the mobile version while working on your desktop. You must have a `list-style-type` somewhere in your media query. Probably just moving out of there will do the trick.

image

**Review by D4nielJ — APPROVED** [review]
## Hello @danalvrz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (danalvrz) [reply styles.css:435]: Ok, I'll do the change, thanks

File: index.html (hunk 2 above)
Comment (danalvrz) [reply index.html:199]: Will fix this too!

**Pull request merged**